show/main.py

```diff
@@ -716,22 +716,59 @@ def pwm_buffer_pool():
 # 'mac' command ("show mac ...")
 #
 
-@cli.command()
+@cli.group(invoke_without_command=True)
+@click.pass_context
 @click.option('-v', '--vlan')
 @click.option('-p', '--port')
 @click.option('--verbose', is_flag=True, help="Enable verbose output")
-def mac(vlan, port, verbose):
+def mac(ctx, vlan, port, verbose):
     """Show MAC (FDB) entries"""
+    if ctx.invoked_subcommand is None:
+        cmd = "fdbshow"
+        if vlan is not None:
+            cmd += " -v {}".format(vlan)
+        if port is not None:
+            cmd += " -p {}".format(port)
+        run_command(cmd, display_cmd=verbose)
+
+@mac.command()
+def aging_time():
+    """Show MAC Aging-Time"""
+    config_db = ConfigDBConnector()
+    config_db.connect()
 
-    cmd = "fdbshow"
+    # Fetching data from config_db for SWITCH
+    switch_table = config_db.get_table('SWITCH')
+    switch_keys = switch_table.keys()
```

Comment:
can we have this displayed from APP db? So even if there is config, it will display the default.